### ISSUE TITLE
Change RCS1010 behaviour for types with three character or less.

### ISSUE THREAD

**Issue by lachlankidson** [JosefPihrt/Roslynator#82]
Some types use less than three characters. One example is generics, where it's not uncommon to see `<T, U>`, `<T1, T2, T3>` etc.

Roslynator currently suggests that code such as `T foo = bar as T;` should use `var` as a simplification.
My suggestion is to not display RCS1010 in such scenarios.

**Comment by JosefPihrt** [comment]
I don't think it is good to change default behaviour of RCS1010 and similar analyzers according to this proposal because it would be a breaking change.

It could be implemented as an optional feature that could be turned on.

Problem is that current Roslyn API does not expose any kind of analyzer settings.
